templates/security/adherent_login.html.twig

```diff
@@ -27,7 +27,12 @@
 
                     <a href="{{ path('forgot_password') }}" class="form__forgotten-pw">Oublié ?</a>
                 </li>
-
+                <li>
+                    <div class="b__nudge--top text--small form__checkbox form__checkbox--large" id="field-conditions">
```

Comment:
The `field-conditions` id is really needed here ?

Reply:
nop, removed 👍